napps/kytos/of_l2ls/main.py

```diff
@@ -51,6 +52,9 @@ def handle_packet_in(self, event):
 
         switch.update_mac_table(ethernet.source, in_port)
 
+        if packet_in.reason != PacketInReason.OFPR_NO_MATCH:
+            return
```

Comment:
Sorry, but I still don' t understand why is it "reducing the funcionality of l2ls"...
Do you mean we should let l2ls send a flow_mod or flood it? because that's what it was doing.